### ISSUE TITLE
beats 7.0.0-alpha2 rpm/deb won't start as a service

### ISSUE THREAD

**Issue by LeeDr** [elastic/beats#9558]
For confirmed bugs, please report:
- Version:  7.0.0-alpha2-2804cede default distribution
- Operating System: CentOS 7
- Discuss Forum URL:
- Steps to Reproduce: 
1. Download and install the rpm file.
1. `service metricbeat start`
1. no log in /var/log/metricbeat
1. check service, it's not running
1. no docs indexed

@tudor found error 
`metricbeat[10181]: Exiting: error loading config file: stat --path.home: no such file or directory`

You can start it in the foreground and it works ok, creates log file.  So there is that work-around.

Packetbeat and Filebeat seem to have the same issue but I haven't done any troubleshooting on them yet.

**Comment by LeeDr** [comment]
Looks like there's the same issue with .deb packages on debian 9 OS;
```
root@packer-virtualbox-iso-1518710300:/home/vagrant# service metricbeat status
● metricbeat.service - Metricbeat is a lightweight shipper for metrics.
   Loaded: loaded (/lib/systemd/system/metricbeat.service; disabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Fri 2018-12-14 18:07:30 UTC; 6min ago
     Docs: https://www.elastic.co/products/beats/metricbeat
  Process: 3895 ExecStart=/usr/share/metricbeat/bin/metricbeat $BEAT_LOG_OPTS $BEAT_CONFIG_OPTS $BEAT_PATH_OPTS (code=exited, status=1/FAILURE)
 Main PID: 3895 (code=exited, status=1/FAILURE)

Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: metricbeat.service: Unit entered failed state.
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: metricbeat.service: Failed with result 'exit-code'.
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: metricbeat.service: Service hold-off time over, scheduling restart.
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: Stopped Metricbeat is a lightweight shipper for metrics..
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: metricbeat.service: Start request repeated too quickly.
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: Failed to start Metricbeat is a lightweight shipper for metrics..
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: metricbeat.service: Unit entered failed state.
Dec 14 18:07:30 packer-virtualbox-iso-1518710300 systemd[1]: metricbeat.service: Failed with result 'exit-code'.
```

**Comment by andrewkroh** [comment]
Fixed by #9562.
